src/js/models/maps/assets/CesiumVectorData.js

```diff
@@ -603,12 +607,21 @@ define([
        * @since 2.25.0
        */
       styleBillboard: function (entity, styles) {
-        if (!this.pinBuilder) {
-          this.pinBuilder = new Cesium.PinBuilder();
-        }
-        entity.billboard.image = this.pinBuilder
-          .fromColor(styles.color, styles.markerSize)
-          .toDataURL();
+        const pin = new DOMParser()
+            .parseFromString(PIN_SVG_STRING, "image/svg+xml")
+            .querySelector("svg");
+        pin.setAttribute("width", styles.markerSize);
+        pin.setAttribute("height", styles.markerSize);
+
+        const pinPath = pin.querySelector("path");
+        pinPath.setAttribute("fill", styles.color.toCssHexString());
+        pinPath.setAttribute("fill-rule", "evenodd");
+        pinPath.setAttribute("stroke", "white");
```

Comment:
Could stroke color be `styles.outlineColor.toCssHexString()` so that it's customizable? 

Similarly, should we respect the `styles.outline` property and make the stroke transparent if it is set to `false`?

Reply:
looks like outlineColor is used for polygon + point, and besides styling, is used to indicate selected vs not selected. the use for the white stroke here is really only for styling purposes- the [free svg](https://fontawesome.com/v6/icons/location-dot?f=classic&s=solid) doesn't have an outline but the [mock](https://www.figma.com/file/9R80DBDAMQtOsu8y1HkAVT/permafrost-discovery-gateway?type=design&node-id=0-3768&mode=design&t=tiKPvHDDeatYKtDr-4) does. idt users need that granular of a control for the pin styling, and I also don't want to opt in to the outline treatment on select. I think hardcode is sufficient here.

Reply:
Really good points @yvonnesjy !